src/boot/efi/secure-boot.h

```diff
@@ -4,5 +4,20 @@
 #include <efi.h>
 #include "efivars-fundamental.h"
 
+typedef enum {
+        ENROLL_OFF,         /* no Secure Boot key enrollment whatsoever, even manual entries are not generated */
+        ENROLL_MANUAL,      /* Secure Boot key enrollment is strictly manual: manual entries are generated and need to be selected by the user */
+        ENROLL_FORCE,       /* Secure Boot key enrollment may be automatic if it is available but might not be safe */
+        _ENROLL_MAX,
+} secure_boot_enroll;
+
+static const CHAR16 * const secure_boot_enroll_table[_ENROLL_MAX] = {
+        [ENROLL_OFF]    = L"off",
+        [ENROLL_MANUAL] = L"manual",
+        [ENROLL_FORCE]  = L"force",
+};
```

Comment:
I think it's better/simpler not to do this lookup table (especially not within a header). Either do it like it's done for console-mode or move the table into `print_status`.

```suggestion
static const char16_t * const secure_boot_enroll_table[_ENROLL_MAX] = {
        [ENROLL_OFF]    = u"off",
        [ENROLL_MANUAL] = u"manual",
        [ENROLL_FORCE]  = u"force",
};
```